wp-admin/edit-tag-form.php

```diff
@@ -143,9 +143,21 @@
 			<td><input name="name" id="name" type="text" value="<?php echo $tag_name_value; ?>" size="40" aria-required="true" aria-describedby="name-description" />
 			<p class="description" id="name-description"><?php echo $tax->labels->name_field_description; ?></p></td>
 		</tr>
+		<?php if ( 'post_format' === $taxonomy ) : ?>
+			<tr class="form-field form-required term-plural-name-wrap">
+				<th scope="row"><label for="plural-name"><?php _ex( 'Plural Name', 'Post Format plural name' ); ?></label></th>
+				<td><input name="plural_name" id="plural-name" type="text" value="<?php echo get_post_format_plural_name( $tag->slug ); ?>" size="40" aria-required="true" aria-describedby="plural-name-description" />
```

Comment:
Make sure to escape the output.

```suggestion
				<td><input name="plural_name" id="plural-name" type="text" value="<?php echo esc_attr( get_post_format_plural_name( $tag->slug ) ); ?>" size="40" aria-required="true" aria-describedby="plural-name-description" />
```